README.md

```diff
@@ -28,13 +28,15 @@ Master and Worker nodes:
 
 ### Preparation
 
-1. Download the latest SUSE CaaS Platform for OpenStack image (for example, SUSE-CaaS-Platform-2.0-OpenStack-Cloud.x86_64-1.0.0-GM.qcow2) from https://download.suse.com.
+1. Download the latest SUSE CaaS Platform for OpenStack image (for example, SUSE-CaaS-Platform-3.0-for-OpenStack-Cloud.x86_64-3.0.0-GM.qcow2)
+   from http://download.suse.de/install/SUSE-CaaSP-3-GM/SUSE-CaaS-Platform-3.0-for-OpenStack-Cloud.x86_64-3.0.0-GM.qcow2
```

Comment:
This link is available only in internal network, please change it to `https://download.suse.com`